edn/src/edn.rustpeg

```diff
@@ -76,25 +76,22 @@ namespace_separator = "/"
 symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>]
 symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>]
 
-symbol_namespace = symbol_char_initial+ (namespace_divider symbol_char_subsequent+)*
+symbol_namespace = symbol_char_initial+ symbol_char_subsequent* (namespace_divider symbol_char_subsequent+)*
```

Comment:
> Shouldn't this be = symbol_char_initial symbol_char_subsequent, with no +?

Yup, my brain intuitively read that as `symbol_char_initial symbol_char_subsequent*`.